rust/lang/security/audit/dangerous-exec-command.yaml

```
@@ -0,0 +1,32 @@
+rules:
+  - id: dangerous-exec-command
+    patterns:
+      - pattern-either:
+          - patterns:
+              - pattern-either:
+                  - pattern: |
+                      Command::new($CMD)
+          - patterns:
+              - pattern-either:
+                  - pattern: |
+                      Command::new("=~/(sh|bash|ksh|csh|tcsh|zsh|ls)/").arg("-c").arg($CMD)
+                  - pattern: |
+                      Command::new("=~/(sh|bash|ksh|csh|tcsh|zsh)/").arg("-c").arg($CMD).output()
+                  - pattern: |
+                      Command::new("=~/(sh|bash|ksh|csh|tcsh|zsh)/").arg("-c").arg($CMD).output().expect(...)
```

```suggestion
          - pattern-either:
              - pattern: |
                  Command::new("=~/(sh|bash|ksh|csh|tcsh|zsh|ls)/").arg("-c").arg($CMD)
              - pattern: |
                  Command::new("=~/(sh|bash|ksh|csh|tcsh|zsh)/").arg("-c").arg($CMD).output()
              - pattern: |
                  Command::new("=~/(sh|bash|ksh|csh|tcsh|zsh)/").arg("-c").arg($CMD).output().expect(...)
```
unnecessary operators